packages/survey-creator-knockout/package.json

```diff
@@ -20,7 +20,7 @@
         "doc:gen": "node doc_generator/editor_docgenerator.js src/entries/index.ts",
         "doc:update": "chmod +x docupdate_npm.sh && ./docupdate_npm.sh",
         "testcafe": "concurrently \"http-server --silent\" \"testcafe -c 4 chrome ../../testCafe/ --selector-timeout 1500\"",
-        "testcafe:file": "concurrently \"http-server --silent\" \"testcafe chrome ../../testCafe/designer/page-edit-modes.ts --reporter minimal --selector-timeout 1500 --debug-mode\"",
+        "testcafe:file": "concurrently \"http-server --silent\" \"testcafe chrome ../../testCafe/designer/page-navigator.ts --reporter minimal --selector-timeout 1500 --debug-mode\"",
         "testcafe:a11y": "concurrently \"http-server --silent\" \"testcafe chrome ./testCafe/accessibility.js --selector-timeout 1500\"",
```

Comment:
Maybe you should not give changes to package.json

Reply:
Why? It doesn't matter.